### PR TITLE
frint-react: className prop in Region component

### LINKED ISSUE #337: frint-react: Support `className` prop in Region component
## Currently

Regions are embedded in Components like this:

```js
<Region name="sidebar" />
```

## Proposed feature

Allow `className` to be passed too like a regular React component:

```js
<Region
  name="sidebar"
  className="region-sidebar"
/>
```

This would allow CSS to target those elements, which may come from external source (outside of the primary application):

```css
.region-sidebar {
  display: none;
}
```

### PR DESCRIPTION
**Pull request by ascariandrea**
I've followed the issue #337  and I have added the `className` to `Region` component.

Closes #337

### CODE REVIEW

File: packages/frint-component-handlers/src/RegionHandler.js [hunk 1]
```diff
@@ -53,9 +53,10 @@ export default {
               return;
             }
 
-            const regionArgs = this.getProp('uniqueKey')
-              ? [this.getProp('name'), this.getProp('uniqueKey')]
-              : [this.getProp('name')];
+            const regionArgs = [
```
Comment (fahad19) [inline packages/frint-component-handlers/src/RegionHandler.js]: I don't think we need to modify the Handlers for this. Just updating `frint-react` package should be enough.

File: packages/frint-react/src/components/Region.js [hunk 2]
```diff
@@ -56,13 +57,15 @@ export default class Region extends React.Component {
...
+
     return (
-      <div>
+      <div className={className}>
         {listForRendering.map((item) => {
           const { Component, name } = item;
 
           return (
-            <Component key={`app-${name}`} />
+            <Component className={className} key={`app-${name}`} />
```
Comment (fahad19) [inline packages/frint-react/src/components/Region.js]: we can remove the `className` prop from here, and keep it only in the parent element.

File: packages/frint-component-handlers/src/RegionHandler.js (hunk 1 above)
Comment (ascariandrea) [reply packages/frint-component-handlers/src/RegionHandler.js]: Ok, this was simpler than I thought.
I've removed the 2 commits related to `frint-component-handlers`

File: packages/frint-react/src/components/Region.js (hunk 2 above)
Comment (ascariandrea) [reply packages/frint-react/src/components/Region.js]: Done!

File: packages/frint-react/src/components/Region.spec.js [hunk 3]
```diff
@@ -353,4 +353,53 @@ describe('frint-react › components › Region', function () {
...
+    // register apps
+    window.app.registerApp(App1, {
+      regions: ['sidebar'],
+      weight: 10,
+    });
+
+
+    // verify
+    const paragraph = document.querySelector('p'); // @TODO: enzyme can be used?
+    expect(paragraph.parentElement.className).to.equal(className);
```
Comment (fahad19) [inline packages/frint-react/src/components/Region.spec.js]: 👍

File: packages/frint-react/src/components/Region.spec.js [hunk 4]
```diff
@@ -353,4 +353,53 @@ describe('frint-react › components › Region', function () {
     // rootApp should not have the instance any more
     expect(window.app.getAppInstance('App', 'todo-item', 'todo-item-1')).to.equal(null);
   });
+
+  it('should accept className and pass down to rendered component', () => {
```
Comment (fahad19) [inline packages/frint-react/src/components/Region.spec.js]: in tests, we usually avoid arrow functions unless they are single-lined, or needs to access parent scope.

can be just `function () {` here.

File: packages/frint-react/src/components/Region.spec.js [hunk 5]
```diff
@@ -353,4 +353,53 @@ describe('frint-react › components › Region', function () {
...
+
```
Comment (fahad19) [inline packages/frint-react/src/components/Region.spec.js]: newline can be removed.

File: packages/frint-react/src/components/Region.spec.js [hunk 6]
```diff
@@ -353,4 +353,53 @@ describe('frint-react › components › Region', function () {
...
+      window.app,
+      document.getElementById('root')
+    );
+
+    // register apps
+    window.app.registerApp(App1, {
+      regions: ['sidebar'],
+      weight: 10,
+    });
+
```
Comment (fahad19) [inline packages/frint-react/src/components/Region.spec.js]: newline can be removed

**Comment by fahad19** [comment]
great stuff, @ascariandrea! many thanks for the PR!

could you kindly also update the API docs in `packages/frint-react/README.md`? we need to document the usage of `className` prop for `Region` component.

**Comment by ascariandrea** [comment]
Sure @fahad19! 
I've added the `className` to available props of `Region`, do you want me to add an example of its usage?
I didn't add the example cause I think it's quite obvious.

**Comment by fahad19** [comment]
@ascariandrea: it's good enough as it is! :)

just merged your PR. I will go for a new release later today or tomorrow most likely.

many thanks again! 💖

**Comment by ascariandrea** [comment]
Many thanks to you guys :)

You gifted me a new tshirt https://hacktoberfest.digitalocean.com/stats/ascariandrea 🎉

**Pull request merged**